static/app/views/issueList/overview.spec.jsx

```diff
@@ -1046,207 +1041,41 @@ describe('IssueList', function () {
   });
 
   describe('transitionTo', function () {
-    let instance;
-    beforeEach(function () {
-      wrapper = shallow(<IssueListOverview {...props} />);
-      instance = wrapper.instance();
-    });
-
-    it('transitions to query updates', function () {
-      instance.transitionTo({query: 'is:ignored'});
-
-      expect(browserHistory.push).toHaveBeenCalledWith({
-        pathname: '/organizations/org-slug/issues/',
-        query: {
-          environment: [],
-          project: [parseInt(project.id, 10)],
-          query: 'is:ignored',
-          statsPeriod: '14d',
-          referrer: 'issue-list',
-        },
-      });
-    });
-
-    it('transitions to cursor with project-less saved search', function () {
```

Comment:
Since this test mocks out the components required to test this, I deleted these. 

I'm also currently working on redoing a lot of the saved search stuff anyway, so these will be unnecessary very soon.